outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "cd_webhook" {
+  value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
```

Comment:
We should [mark this output as sensitive](https://www.terraform.io/docs/language/values/outputs.html#sensitive-suppressing-values-in-cli-output) since it contains a password - that way Terraform will avoid logging it unless explicitly told to